### DISCUSSION TITLE
Support utils for the bloom filter

### DISCUSSION THREAD

**Discussion by ruXlab** [web3j/web3j#1795] [Ideas]
## Feature description_

Ethereum has very powerful tool - bloom filter, with it many potentially unnecessary operations can be avoided. For instance, when checking for specific logs in the block no need to pull all transactions of it. Instead the bloom filter match can be checked first on the client side and then if there is a match the complete block information can be retrieved.

It's just one of the use cases

[web3js](https://web3js.readthedocs.io/en/v1.2.4/web3-utils.html#bloom-filters) has these utility functions

**Comment by mohamedelshami** [comment]
Thanks @ruXlab, this is an interesting suggestion. I'm moving this to discussions / Ideas and hopefully, someone can add this feature to Web3j at some point. Also please feel free to introduce this feature if you have time for it.

**Reply by mhrsalehi** [reply]
Hi, I Implemented this feature. 
https://github.com/web3j/web3j/pull/1932.

**Comment by mhrsalehi** [comment]
this feature was merged to master branch in https://github.com/web3j/web3j/pull/1932
